Dockerfile

```diff
@@ -46,7 +46,7 @@ RUN pip install -U pip && \
                 cloudscraper && \
     rm -rf /wheels
 
-COPY --from=0 /flexget-ui-v2 /usr/local/lib/python3.10/site-packages/flexget/ui/v2/
+COPY --from=0 /flexget-ui-v2 /usr/local/lib/python3.11/site-packages/flexget/ui/v2/
```

Comment:
should move this line right after `FROM` so next time we'll notice it.

Reply:
Or add a warning comment about it before `FROM`

Reply:
I wonder if we can switch it to call the `dev_tools.py bundle-webui` command like the release script does rather than doing the copy manually. That would mean we don't have to remember anymore.

Reply:
Maybe create a TODO issue and do it later😁

Reply:
I decided to give that method a test and opened up a separate PR that more accurately describes the change. Will leave that one open for a minute in case anyone notices issues with that implementation.